src/Client.js

```diff
@@ -0,0 +1,75 @@
+var wrapper = require('./GitWrapper.js');
+// set up yargs command line parsing
+var argv = require('yargs')
+  .usage('Usage: $0 <command> [options]')
+  .command('add_team', 'Add a team to an organization')
+  .command('add_repo_to_team', 'Add a repo to a team')
+  .command('invite_user', 'Invite a user to a team')
+  .command('remove_user', 'Remove a user from a team')
+  .command('rename_team', 'Renames a team within the organization')
+  .example('$0 add_team -T \'123456789abcdefghi\' -t sample -o eclipse-test', 'Add a team of name sample to the eclipse-test organization')
```

Comment:
Never read token/secret/password from the command line. This is how secrets are leak (can be observed via a simple  `ps` or `top`). Better read secrets from stdin, this way the token can be stored in a file and `cat`ed and piped to the CLI. 

cat file_with_secret | sync_script.js add_team -t sample -o eclipse-test

I know that most of the time the script will run on a "private" machine, but better safe than sorry.